pkg/api/types.go

```diff
@@ -64,30 +76,30 @@ const (
 
 type ConditionStatus string
 
-// These are valid condition statuses. "ConditionTrue" means a resource is in the condition;
-// "ConditionFalse" means a resource is not in the condition; "ConditionUnknown" means kubernetes
-// can't decide if a resource is in the condition or not. In the future, we could add other
-// intermediate conditions, e.g. ConditionDegraded.
+// These are valid condition statuses. "ConditionTrue" means a resource is in
+// the condition; "ConditionFalse" means a resource is not in the condition;
+// "ConditionUnknown" means kubernetes can't decide if a resource is in the
+// condition or not. In the future, we could add other intermediate
```

Comment:
Do we have examples of "ConditionalUnkown"?  I'm just curious to know of a case where Kube wouldn't know if something has happened or not (or it shouldn't assume that its "False")